packages/ui-library/src/components/textarea/index.stories.ts

```diff
@@ -36,12 +36,12 @@ const defaultParams: BlrTextareaType = {
   labelAppendix: '(Appendix)',
   hasHint: false,
   hintMessage: 'This is a small hint message',
-  hintIcon: 'blrInfo',
+  hintMessageIcon: 'blrInfo',
   arialabel: 'Text Area',
   name: 'Text Area',
-  showCounter: true,
+  hasCounter: true,
   theme: 'Light',
-  textareaId: '#textAreaId',
+  textAreaId: '#textAreaId',
```

Comment:
I this naming for TextArea. textAreaId seems ok to me, but does this mean that we have to change it everywhere in the app? For example, we may need to update the render function: https://github.com/deven-org/boiler/blob/develop/packages/ui-library/src/components/textarea/renderFunction.ts#L6, the label in our example JS app: https://github.com/deven-org/boiler/blob/develop/packages/js-example-app/src/index.ejs#L79 and the component type in index.ts: https://github.com/deven-org/boiler/blob/develop/packages/ui-library/src/components/textarea/index.ts#L256

Reply:
Hey @davidken91 , hey @JpunktWpunkt , the goal should be to be very consistent with our component names. 

In the documentation we are writing our components in Title Case, so it is Text Area. In code we decided to go with lowerCamelCase, so textArea would be correct. However we are looking into changing this with https://github.com/deven-org/boiler/issues/970 and to use kebab-case instead. Part of this issue is also to clearly define where to use which typecase.

Therefor I would say we should not update the examples you have linked and wait for #970.